src/Definition.php

```diff
@@ -95,4 +96,31 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Gets the definitons of all ancestor classes
+     *
+     * @return Definition[]
```

Comment:
I was wondering what to use here - ```\Generator``` or ```\Iterator```, ideally it should be something indicating both the type of the yielded values, like ```\Iterator[Definition]``` but I do not think this is supported. I will add ```\Generator``` for now.